src/crab/ebpf_domain.cpp

```diff
@@ -2069,8 +2070,10 @@ void ebpf_domain_t::do_store_stack(NumAbsDomain& inv, const number_t& width, con
     std::optional<variable_t> var = stack.store_type(inv, addr, width, val_type);
     type_inv.assign_type(inv, var, val_type);
     if (width == 8) {
-        inv.assign(stack.store(inv, data_kind_t::svalues, addr, width, val_svalue), val_svalue);
-        inv.assign(stack.store(inv, data_kind_t::uvalues, addr, width, val_uvalue), val_uvalue);
+        auto slot_svalue = stack.store(inv, data_kind_t::svalues, addr, width, val_svalue);
+        auto slot_uvalue = stack.store(inv, data_kind_t::uvalues, addr, width, val_uvalue);
+        inv.assign(slot_svalue, val_svalue);
+        inv.assign(slot_uvalue, val_uvalue);
```

Comment:
This change is crucial. Without it, the svalue of the slot is removed during the second store. I don't know why it did not surface before.